uwreact_bringup/launch/zeds.launch

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0"?>
+
+<launch>
+    <arg name="zed_id1"               default="0" />
+
+    <arg name="pose_frame"            default="map" />
+    <arg name="camera_model"          default="0" /> <!-- 0=ZED, 1=ZEDM-->
+    <arg name="serial_number"         default="0" />
+    <arg name="verbose"               default="true" />
+
+    <arg name="resolution"            default="2" /> <!--0=RESOLUTION_HD2K, 1=RESOLUTION_HD1080, 2=RESOLUTION_HD720, 3=RESOLUTION_VGA -->
+    <arg name="frame_rate"            default="30" />
+    <!-- RESOLUTION_HD2K   -> 2208*1242, available framerates: 15 fps.
+         RESOLUTION_HD1080 -> 1920*1080, available framerates: 15, 30 fps.
+         RESOLUTION_HD720  -> 1280*720,  available framerates: 15, 30, 60 fps.
+         RESOLUTION_VGA    -> 672*376,   available framerates: 15, 30, 60, 100 fps. -->
+
+    <arg name="depth_topic"           default="depth/depth_registered" />
+    <arg name="depth_cam_info_topic"  default="depth/camera_info" />
+    <arg name="laserscan_topic"       default="depth/laserscan"/>
+
+    <arg name="base_frame"            default="zed_camera_center"/>
+    <arg name="laserscan_height"      default="10"/>
+    <arg name="range_min"             default="0.45"/>	
+
+    <!-- First ZED camera -->
+    <group ns="zed1">
```

Comment:
Should maybe give this a more descriptive name. zed_left? And match it with the zed id Arg name

Reply:
I was thinking we'd wait till we know where it is on the robot, then give it more descriptive names, but I can put a placeholder for now

Reply:
I can add a TODO to update them when there placement is more well known?

Reply:
We can definitely wait until later to give them better names, but the namespace and zed_id should match each other.

Reply:
I was wrong - Didn't realize zed_id was an integer device ID as assigned by the OS, thought it was just a string identifier. Ignore the previous comment about making the namespace and zed_id match. Melvin is still correct in that the namespace should be given a more descriptive name once we know where it will be placed.